web/themes/custom/recipe_theme/templates/views/views-view-unformatted--catalog.html.twig

```diff
@@ -0,0 +1,34 @@
+{#
+/**
+ * @file
+ * Theme override to display a view of unformatted rows.
+ *
+ * Available variables:
+ * - title: The title of this group of rows. May be empty.
+ * - rows: A list of the view's row items.
+ *   - attributes: The row's HTML attributes.
+ *   - content: The row's content.
+ * - view: The view object.
+ * - default_row_class: A flag indicating whether default classes should be
+ *   used on rows.
+ *
+ * @see template_preprocess_views_view_unformatted()
+ */
+#}
+{% if title %}
+  <h3>{{ title }}</h3>
+{% endif %}
+<div class="row">
+
+{% for row in rows %}
+  {%
+    set row_classes = [
+      default_row_class ? '',
+      'col-4'
```

Comment:
This line means you'll have a three column layout on every screens. Are you sure this also looks good on mobil?